### ISSUE TITLE
Image Separater

### ISSUE THREAD

**Issue by vipinkatara** [python-geeks/Automation-scripts#413]
## Script Title - Image Separater

**Brief** - The script will allow a user to separate different type of images from a directory based on their feature vectors.

## Instructions

- Create a new folder for your script and file/folder name should be appropriate.
- Create a README.md in your folder for program Instructions
- add requirements.txt if needed

### Programming Language

- [x] Python

Happy Coding

**Comment by vipinkatara** [comment]
Please assign this issue to me.

**Comment by pawangeek** [comment]
Can you explain a bit more, Like I have dog and cat images in folder it will separate out both?

**Comment by vipinkatara** [comment]
Yes, it'll separate out both dog and cat images if we choose cluster parameter as 2. similarly it can also separate humans, animals, tools, etc. But we have define in how many different categories we have to separate manually.
